### PR TITLE
fix(title): Fixed #968.

### PR DESCRIPTION
**Pull request by michaelkleinhenz**
Fixed https://github.com/openshiftio/openshift.io/issues/968, special characters in title strings are now displayed correctly in the list entry, the board view and quick and full preview views.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

File: src/app/components/card/card.component.html [hunk 1]
```diff
@@ -48,9 +48,7 @@
         tooltip="unassigned"
         placement="top"></span>
     </div>
-    <div class="margin-right-15 padding-right-15" tooltip="{{cardValue.title}}" placement="top">
-      {{ cardValue.title | truncate : 80 }}
-    </div>
+    <div class="margin-right-15 padding-right-15" tooltip="{{cardValue.title}}" placement="top" [innerHTML]="cardValue.title | truncate : 80"></div>
```
Comment (sanbornsen) [inline src/app/components/card/card.component.html]: Can you maintain the indentation and line break? something like this - 
```
<div class="margin-right-15 padding-right-15" 
        tooltip="{{cardValue.title}}" 
        placement="top" 
        [innerHTML]="cardValue.title | truncate : 80">
</div>
```

File: src/app/components/work-item-detail/work-item-detail.component.html [hunk 2]
```diff
@@ -78,8 +78,8 @@ <h3 *ngIf="workItem.id" class="detail-id" id="wi-detail-id">{{workItem.attribute
                 class="posRel"
                 *ngIf="!headerEditable" (click)='openHeader()'
                 id="title-click-div">
-                <h2 *ngIf="!loggedIn" id="wi-detail-title-ne" class="detail-title truncate margin-top-15 form-control-static">{{ workItem.attributes['system.title'] }}</h2>
-                <h2 *ngIf="loggedIn && workItem.id" id="wi-detail-title-click" class="detail-title truncate margin-top-15">{{ workItem.attributes['system.title'] }}</h2>
+                <h2 *ngIf="!loggedIn" id="wi-detail-title-ne" class="detail-title truncate margin-top-15 form-control-static" [innerHTML]="workItem.attributes['system.title']"></h2>
```
Comment (sanbornsen) [inline src/app/components/work-item-detail/work-item-detail.component.html]: Same with these two lines here.

**Comment by sanbornsen** [comment]
This is amazing, I never realised this could solve the special character problem 👍

File: src/app/components/card/card.component.html (hunk 1 above)
Comment (michaelkleinhenz) [reply src/app/components/card/card.component.html]: Done.

File: src/app/components/work-item-detail/work-item-detail.component.html (hunk 2 above)
Comment (michaelkleinhenz) [reply src/app/components/work-item-detail/work-item-detail.component.html]: This section was replaced anyway.

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

**Comment by nimishamukherjee** [comment]
Fixes https://openshift.io/openshiftio/openshiftio/plan/detail/1524

**Comment by rgarg1** [comment]
The issue is fixed. Thanks @michaelkleinhenz 

special chars fixed

**Comment by rgarg1** [comment]
Ah! Two more place where the fix needs to go:

1) Workitem's link summary section (see snapshot)
2) The green popup alert message

escaped chars in preview screen

**Comment by michaelkleinhenz** [comment]
@rgarg1 which green popup you mean?

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-7 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-8 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

**Comment by nimishamukherjee** [comment]
@rgarg1 @jarifibrahim please note that the notification popup is outside the scope of Planner.
You can file an issue on ngx-base. This is where the component resides.

**Comment by rgarg1** [comment]
@michaelkleinhenz The link panel is still broken. Have a look:

for_michael

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-9 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

**Comment by michaelkleinhenz** [comment]
@rgarg1 @jarifibrahim updated.

**Comment by fabric8cd** [comment]
@michaelkleinhenz fabric8/fabric8-ui:SNAPSHOT-PR-2169-10 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2169-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
Fixed!

**Pull request merged**